roles/pulp/templates/pulp.container.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Pulp 
+After=network.target
+
+[Container]
+Image=pulp/pulp:stable
+ContainerName=pulp
+PublishPort=8080:80
+Volume=/var/lib/pulp/settings:/etc/pulp:Z
+Volume=/var/lib/pulp/pulp_storage:/var/lib/pulp:Z
+Volume=/var/lib/pulp/pgsql:/var/lib/pgsql:Z
+Volume=/var/lib/pulp/containers:/var/lib/containers:Z
```

Comment:
If you don't care about where the volume is on the host system you can also use relative paths:
```suggestion
Volume=pulp_settings:/etc/pulp
Volume=pulp_storage:/var/lib/pulp
Volume=pulp_pgsql:/var/lib/pgsql
Volume=pulp_containers:/var/lib/containers
```

Benefit is that you don't need to ensure the directory (simplifying your Ansible file). Downside is that you can't just take a backup of `/var/lib/pulp`. On the other hand, you may want to approach backups in a different way anyway.